### ISSUE TITLE
Direct option to create private note on contacts

### ISSUE THREAD

**Issue by j-hellenberg** [hpi-swt2/contacts-portal#33]
As a user, I can only see notes that I have created for other users.
- [ ] In the notes list, I can only see notes that I have created for other users
- [ ] No one else can see these notes

Deferred because of unmet dependencies:
- add button in contact list (depends on #26)
- display note on profile of that user (depends on #30)
